showcase/package-lock.json

```diff
@@ -1524,8 +1524,7 @@
       "integrity": "sha512-flCoA7F4wRkymaP6zwAWPrgqDLx4eQM5Ep/pLtmMyVp/FQz0eevwyNfa4lEaYSFHzcs2bKVJr8O35JrioQL/Yg=="
     },
     "@nationalbankbelgium/stark-build": {
-      "version": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.0.0-rc.2-0c7801b2.tgz",
-      "integrity": "sha512-GcmIdKEVH95WNRt9n/jRBQez7HpGIFF5aXPnMt5k+nkXJObQXc1VQyUZH7+deCOFwhgij7UwBAHpi7hq/ZpdAw==",
+      "version": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.0.0-rc.2-c99475f8.tgz","integrity": "sha512-CEY55bmM9ECWfVUFgP/CZ2y/ab4TzfHQD+WQC1H2b2mE8sSm1yyOkXNbl71wM/ql6z0czfpFTK45SXvOHUcNEw==",
```

Comment:
Could you please revert the changes in this file since they are not necessary ? 😊